### PR TITLE
fixing two issues

### PR DESCRIPTION
**Pull request by ghost**
bug fixes: 
- removed excessive logging 
- fixed precommit hook as it was reading parameters as 1 single string

### CODE REVIEW

**Review by balloonpopper — APPROVED** [review]

**Review by micklove — COMMENTED** [review]
No surprises?
Can we put in a usage command instead? (which takes 0 or 1 as a param and exits at the end)

File: lib/githooks/pre-commit [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/bash
 # shellcheck disable=SC2039
```
Comment (micklove) [inline lib/githooks/pre-commit:9]:
```suggestion
usage()
{
  echo "Usage : $0 <files>"
  echo "${2}" # The error message
  exit "${1:-0}"
}

```

File: lib/githooks/pre-commit [hunk 2]
```diff
@@ -64,16 +64,16 @@ determine_git_revision
 RC=0
 
 # Ensure we are in the top level of the git repo
-cd "$(git rev-parse --show-toplevel)"
+cd "$(git rev-parse --show-toplevel)" || exit 1
```
Comment (micklove) [inline lib/githooks/pre-commit]:
```suggestion
cd "$(git rev-parse --show-toplevel)" || usage "You must run in a .git enabled folder" 1
```

File: lib/githooks/pre-commit [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/bash
```
Comment (micklove) [inline lib/githooks/pre-commit]:
```suggestion
#!/bin/bash -e
```

**Review by micklove — CHANGES_REQUESTED** [review]

File: lib/githooks/pre-commit (hunk 3 above)
Comment (ghost) [reply lib/githooks/pre-commit]: What's the significance of this? vs set -e?

**Review by micklove — APPROVED** [review]
Noice work

**Pull request merged**